dtfs-central-api/src/cron-scheduler-jobs/deal-cancellation/cancel-deal-job.test.ts

```diff
@@ -53,15 +53,15 @@ describe('cancelDealJob', () => {
     expect(cancelDealJob.description).toEqual('Cancel deals in the database that are pending cancellation & the effective from date has passed');
   });
 
-  it('it calls findPendingDealCancellations', async () => {
+  it('calls findPendingDealCancellations', async () => {
```

Comment:
can we use `should call .... `here please

Reply:
Literally just done this 😊

Reply:
Aligned all test descriptions in this file.